src/main/java/io/cdap/plugin/servicenow/apiclient/ServiceNowTableAPIClientImpl.java

```diff
@@ -76,10 +79,12 @@ public class ServiceNowTableAPIClientImpl extends RestAPIClient {
   private static final String GLIDE_DATE_TIME_DATATYPE = "glide_date_time";
   private static final Gson GSON = new Gson();
   private final ServiceNowConnectorConfig conf;
+  public final Boolean useConnection;
   public static JsonArray serviceNowJsonResultArray;
 
-  public ServiceNowTableAPIClientImpl(ServiceNowConnectorConfig conf) {
+  public ServiceNowTableAPIClientImpl(ServiceNowConnectorConfig conf, Boolean useConnection) {
     this.conf = conf;
+    this.useConnection = useConnection;
```

Comment:
Since this class is an implementation, is should be tied to a Single SchemaType.

I do not see the use of  `useConnection` for any thing else. 

So, instead of storing and recomputing again n again, Can we just store `SchemaType` as final field?